projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -373,7 +373,7 @@ public function post_page_metabox_assets() {
 	$( document ).ready( function() {
 		// If we have the #pub-connection-tests div present, kick off the connection test
 		if ( $( '#pub-connection-tests' ).length ) {
-			publicizeConnTestStart();
+			// publicizeConnTestStart(); TODO: Fix magic links
```

Comment:
Sorry for not being clear on this! The connection tester function returns a link for each reparable connection which directly directs you to Facebook for example to fix the connection. These links are broken now. 
More: p1661505891955819-slack-C02JJ910CNL

I commented it out, so while it is not working it is not reaching out for the API introducing some extra calls, however as discussed if they would work it would be nice that the user could solve the issue right from the editor, so after looking up why are they broken and fixing them, we could reenable and refactor this.

Reply:
No issues, I appreciate the explanation, maybe we can add your explanation as the comment? :)